src/app/components/Navbar/Navbar.tsx

```diff
@@ -7,16 +7,10 @@ type Props = {
 
```

Comment:
Best reviewed in "split mode". I actually added a new wrapper `div` to constrain the content's width.
Since I also removed wrapper `div`s from `<AccountMenu>`, `children` and `UserMenu` it's a bit hard to see the changes in other views.